moodle/pkg/apis/moodlecontroller/v1/types.go

```diff
@@ -31,6 +31,8 @@ type MoodleSpec struct {
 	MySQLUserPassword string `json:"mySQLUserPassword"`
 	//Moodle Admin Email
 	MoodleAdminEmail string `json:"moodleAdminEmail"`
+	//PVC Volume Name
+	PVCVolumeName string `json:"PVCVolumeName"`
```

Comment:
I think base Kubernetes objects have a way to tag optional attributes.

Reply:
Ok

Reply:
I'll make a note of this for next time